OmiseSwift/API Models/BankStatus.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  BankStatus.swift
+//  OmiseSwift
+//
+//  Created by ponpol on 1/3/2564 BE.
+//  Copyright © 2564 BE Omise. All rights reserved.
+//
+
+import Foundation
+
+public struct BankStatus: Codable, Hashable {
+   public let code: String
+   public let name: String
+   public let active: Bool
+
+   private enum CodingKeys: String, CodingKey {
```

Comment:
You don't need to add `CodingKeys`, `init(from decoder: Decoder)` and `encode(to encoder: Encoder)` here manually. You can completely remove it. Since the variable names exactly match the keys in the json, swift is able to map everything automatically.